docs/introduction/preparing-for-ci.md

```diff
@@ -172,9 +172,10 @@ detox_e2e:
     - mkdir -p /root/.android && touch /root/.android/repositories.cfg
     # The Dockerimage provides two paths for sdkmanager and avdmanager, which the defaults are from $ANDROID_HOME/cmdline-tools
     # That is not compatible with the one that Detox is using ($ANDROID_HOME/tools/bin)
-    - echo yes | $ANDROID_HOME/tools/bin/sdkmanager --channel=0 --verbose "system-images;android-27;default;x86_64"
+    - echo yes | $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --channel=0 --verbose "system-images;android-27;default;x86_64"
```

Comment:
```suggestion
    - echo yes | $ANDROID_HOME/cmdline-tools/latest/bin/sdkmanager --channel=0 --verbose "system-images;android-27;default;x86_64" "emulator"
```

Reply:
i see, thankyou for your suggestion. will fix ASAP

Reply:
updated @noomorph